src/phrases/phrases-nb.ts

```diff
@@ -770,11 +777,14 @@ const phrases = {
     },
     concept: {
       create: 'Opprett ny forklaring',
+      addText: 'Velg en forklaring under og trykk "koble til fagtekst", eller',
```

Comment:
Kvar kjem dette fra?

Reply:
Stammer fra en merge conflict med phrases når jeg tok inn fra master og det var endringer i phrases både i master og i egen branch. Da måtte jeg legge disse til for å få diffen fin. Nå ser jeg at de har blitt tillegg (som naturligvis ikke er meninga). Har prøvd å fjerne dem igjen.